js/libraries/jbox/jBox.css

```diff
@@ -19,7 +19,6 @@
 .jBox-container {
 	background: #fff;
 	border:1px solid #ffbb00;
-	max-width:180px;
```

Comment:
I added this before as I build the tooltip function a while ago. But i realized now it was not the best place to maximize the width of the tooltip. I added this in the main.css now.